tests/conftest.py

```diff
@@ -27,3 +27,30 @@ def assert_render_template(template, context, expected, request=None):
         assert template.render(context, request) == expected
 
     return assert_render_template
+
+
+@pytest.fixture
+def assert_parse_error():
+    """
+    A convenient method to test `TemplateSyntaxError` for both engines.
+
+    Example:
+        def test_error(assert_parse_error):
+            assert_parse_error(
+                template=...,
+                django_message="invalid literal for int() with base 10: '-5.5'",
+                rusty_message="  × Couldn't convert argument (-5.5) to integer..."
+            )
+
+    """
+
+    def _assert_parse_error(template, django_message, rusty_message):
+        with pytest.raises(TemplateSyntaxError) as exc_info:
+            engines["django"].from_string(template)
+        assert str(exc_info.value) == django_message
+
+        with pytest.raises(TemplateSyntaxError) as exc_info:
+            engines["rusty"].from_string(template)
+        assert str(exc_info.value) == rusty_message
+
+    return _assert_parse_error
```

Comment:
I think this would work:

```suggestion
@pytest.fixture(params=["rusty", "django"])
def assert_parse_error(engine):
    """
    A convenient method to test `TemplateSyntaxError` for both engines.

    Example:
        def test_error(assert_parse_error):
            assert_parse_error(
                template=...,
                django_message="invalid literal for int() with base 10: '-5.5'",
                rusty_message="  × Couldn't convert argument (-5.5) to integer..."
            )

    """

    def _assert_parse_error(template, django_message, rusty_message):
        message = django_message if engine == "django" else rusty_message
        with pytest.raises(TemplateSyntaxError) as exc_info:
            engines[engine].from_string(template)
        assert str(exc_info.value) == message

    return _assert_parse_error
```